src/main/resources/schema/elytron-client-1_2.xsd

```diff
@@ -73,6 +73,7 @@
         </xsd:annotation>
         <xsd:all>
             <xsd:element name="key-store-ssl-certificate" type="key-store-ref-type" minOccurs="0"/>
+            <xsd:element name="key-manager" type="key-manager-type" minOccurs="0"/>
```

Comment:
It should be possible to declare alternative elements, see
https://stackoverflow.com/questions/11508147/xsd-choice-inside-all